iOS Example Frame/Podfile.lock

```diff
@@ -1,9 +1,9 @@
 PODS:
   - CheckoutEventLoggerKit (1.1.1)
-  - Frames (3.5.0):
+  - Frames (3.5.1):
```

Comment:
Should this be Frames 3.5.2 ?

Reply:
These sample projects are just linked to the files in the current repo. This shouldn't be an issue